lib/course_planner_web/templates/offered_course/teacher_edit_form.html.eex

```diff
@@ -0,0 +1,83 @@
+  <div class="row">
+    <div class="
+      col-xs-12
+      col-sm-offset-1 col-sm-10
+      col-md-offset-2 col-md-8
+      col-lg-offset-2 col-lg-8
+    ">
+      <div class="row middle-xs page-header">
+        <div class="col-xs-6 col-sm-9 col-md-10 page-title">
+          <%= Enum.join [@offered_course.course.name, @offered_course.term.name], " - " %>
+        </div>
+        <%= if @user_role == "Coordinator" do %>
```

Comment:
Why is there a coordinator check here if it's a edit form specifically for the teacher?

Reply:
done